### PR TITLE
New Starcraft 2 Cheat Sheet

### PR DESCRIPTION
**Pull request by jonk1993**
**What does your Instant Answer do?**

It is a cheat sheet that includes the cheat codes for all expansions of Starcraft 2.

**What problem does your Instant Answer solve (Why is it better than organic links)?**

Providing the cheat codes in a cheat sheet saves the user the time of finding the correct link and loading an external page to view cheat codes. The user will immediately be shown the cheat codes after querying DuckDuckGo.

**What is the data source for your Instant Answer? (Provide a link if possible)**

The data source is the official Blizzard site (http://us.battle.net/sc2/en/blog/9135373/game-guide-starcraft-ii-cheat-codes-3-1-2012). 

**Why did you choose this data source?**

I chose this data source because it was contained in a post by Blizzard on the official starcraft 2 forums.

**Are there any other alternative (better) data sources?**

There are some alternatives, but the descriptions are typically not as clear and contained some out of date cheat codes.

**What are some example queries that trigger this Instant Answer?**

"Starcraft 2 cheat sheet", "sc2 cheat sheet", "starcraft 2 help"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

This would be particularly useful for gamers.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

No

**Which existing Instant Answers will this one supercede/overlap with?**

None

**Are you having any problems? Do you need our help with anything?**

Nope

**Where did you hear about DuckDuckHack? (For first time contributors)**

Saw it was included in safari! 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screen shot 2015-11-28 at 9 16 52 pm

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 10
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 11
        - [] Microsoft edge
        
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Linux
        - [] Google Chrome
        - [] Firefox
        
    - Mac OSX
        - [X] Google Chrome
        - [X] Firefox
        - [X] Opera
        - [X] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```
---
https://duck.co/ia/view/starcraft2_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jonk1993 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by jophab** [comment]
@jonk1993 I think Name should not contain the word cheat sheet.
Also one of your aliases is same as the file name.

**Comment by jonk1993** [comment]
@jophab thanks for the catch, I addressed your comments.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Starcraft 2 Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/starcraft2_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/starcraft2.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "starcraft2_cheat_sheet",
+    "name": "Starcraft 2",
+    "description": "Cheat codes for the campaign and offline custom games of Starcraft 2",
+    "metadata": {
+        "sourceName": "Battle.net Cheats",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/starcraft2.json]: nit: let's change this to `Battle.net` to reflect the domain name

**Comment by moollaza** [comment]
@jonk1993 this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**